examples/jdbc-complex-type-queries/jdbc_complex_type_queries.bal

```diff
@@ -1,5 +1,5 @@
 import ballerina/io;
-import ballerina/jdbc;
+import ballerina/java.jdbc as jdbc;
```

Comment:
@daneshk Shouldn's this be `ballerinax/java.jdbc`?

Reply:
Yes. This should be `ballerinax/java.jdbc`.